MapleServer2/Types/Player.cs

```diff
@@ -162,20 +162,36 @@ public static Player MaleDefault(long accountId, long characterId, string name =
             };
         }
 
-        public static Player NewCharacter(byte gender, Job jobType, string name, SkinColor skinColor, object equips)
-        {
+        public static Player NewCharacter(byte gender, /*Job jobType,*/ int job, string name, SkinColor skinColor, object equips) {
+            PlayerStats stats = PlayerStats.Default();
+            stats.Hp = new PlayerStat(1000, 0, 1000);
+            stats.CurrentHp = new PlayerStat(0, 1000, 0);
+            stats.Spirit = new PlayerStat(100, 100, 100);
+            stats.Stamina = new PlayerStat(120, 120, 120);
+            stats.AtkSpd = new PlayerStat(120, 100, 130);
+            stats.MoveSpd = new PlayerStat(110, 100, 150);
+            stats.JumpHeight = new PlayerStat(110, 100, 130);
+
             var player = new Player
             {
+                AccountId = 0x1111111111111111,
```

Comment:
I don't know much about MS2's account ids, but there is a good chance that duplicate AccountIds are going to cause issues. We probably want to generate a new one for each account (I'm aware that the db isn't done at this point, if this is a placeholder, it might be worth leaving a TODO comment here). 

The same applies for other placeholder values we might be using right now, we don't want to forget about them.